calculadora.py

```diff
@@ -0,0 +1,32 @@
+class Calculadora:
+
+    def __init__(self, num1, num2):
+        self.valor_a = num1
+        self.valor_b = num2,
+
+    def soma(self):
+        if self.valor_a == 1:
+            return "Valor inválido"
+        elif self.valor_a == 2:
```

Comment:
Criar uma lista de valores inválidos e verificar se o valor_a está na lista. Caso esteja, retornar "Valor inválido". Exemplo:

INVALID_VALUES = [1, 2, 3, 4, 5]
if self.valor_a in INVALID_VALUES:
return "Valor inválido"'}